frontend/src/models/dashboardsModel.tsx

```diff
@@ -92,7 +92,7 @@ export const dashboardsModel = kea<dashboardsModelType>([
                         // If user is anonymous (i.e. viewing a shared dashboard logged out), don't load authenticated stuff
                         return { count: 0, next: null, previous: null, results: [] }
                     }
-                    return await api.get(url || `api/projects/${teamLogic.values.currentTeamId}/dashboards/?limit=100`)
```

Comment:
fly by have you tried this API call change in prod? 

is it still ~2 secs for 400 rows or does it make it slower?

Reply:
(i'd personally change one thing at a time here or make the page size configurable so you can test once the index is out - otherwise we don't know (for sure) what works and what doesn't)

Reply:
Good point... 400 currently takes 4x as long... so touching this in follow up if appropriate.